src/extensions/breeze_carbon.cr

```diff
@@ -5,11 +5,17 @@ require "../breeze_carbon/actions/**"
 require "../breeze_carbon/pages/**"
 
 module BreezeCarbon
+  extend Breeze::Extension
+
   Habitat.create do
     setting email_previews : Carbon::EmailPreviews.class, example: "Emails::Previews"
   end
-end
 
-Breeze::NavbarLinks.settings.links << ->(navbar : Breeze::NavbarLinks) {
-  navbar.mount_link("Emails", to: BreezeCarbon::Emails::Index)
```

Comment:
That thought occurred to me as well, kind of. I could change the module to have one method that must return a `Breeze::NavbarLink`. What do you think about that?

Reply:
Ok, yeah. Lets see what that might look like. It would make sense that we don't want people to throw images and iframes in there, but they also might want the ability to handle roles or other logic. So as long as the end result is a NavbarLink that is following some convention, then it'll probably be fine.